towhee/hparam/hyperparameter.py

```diff
@@ -286,7 +286,7 @@ def put(self, name: str, value: Any):
             obj = obj[p]
         obj[path[-1]] = safe_numeric(value)
 
-    def get(self, name: str) -> Any:
+    def get(self, name: str, default: any = None) -> Any:
```

Comment:
We'd better use get_or_else in the `_Accessor`, and is there a unit test to check the behavior of the parameter `default`?

Reply:
request to change @junjiejiangjjj